add installation command for sudo-free environments (#112)
ZupIT/docs-ritchie: content/en/Getting Started/Macos.md

curl -fsSL https://commons-repo.ritchiecli.io/install.sh | bash
```

> _Note: To install Ritchie on servers or Docker containers (without `sudo` user), use the following command:_

Reviewer: ```suggestion
_Note: To install Ritchie on servers or Docker containers (without `sudo` user), use the following command:_
```
Essa nota, acredito ser melhor colocar como alert de info e remover o alert de info do -If you prefer, you also can follow with the **manual installation**.

curl --output ~/Desktop/Ritchie-CLI-macos-installer-x64.pkg --location https://commons-repo.ritchiecli.io/latest/Ritchie-CLI-macos-installer-x64.pkg
```

## Step 2: Verify installation

> You can confirm if your installation went well by running this command:

Reviewer: ```suggestion
Confirm if your installation worked,  run the command below:
```
